Resolve some minor diffs that have crept in from bad syncs.

Gregable opened the pull request:
The big change is adding validator-amp-ad.html test case. This looks like it was accidentally deleted in #29877.

The other big change is deleting the unused css-selectors.js, removed in cl/278402306 but failed to be synced.

The other changes are dup lines that crept in from an incorrect reverse diff application.

twifkak commented:
The other big change is deleting the unused css-selectors.js, removed in cl/278402306 but failed to be synced.

Gregable commented:
> The other big change is deleting the unused css-selectors.js, removed in cl/278402306 but failed to be synced.

Good catch. I added this to the first comment for clarity.
